src/libpriv/rpmostree-postprocess.c

```diff
@@ -1634,10 +1669,6 @@ rpmostree_prepare_rootfs_for_commit (int            src_rootfs_dfd,
       }
   }
 
-  /* And call into the final postprocessing function */
-  if (!postprocess_final (target_rootfs_dfd, treefile,
```

Comment:
2:  what might be the benefit for splitting those two functionalities apart here? It seems like for the `rpm-ostree compose commit` command, we will no longer call this `rpmostree_prepare_rootfs_for_commit()` function to initialize the rootfs. But we still call this postprocess_final() in the commit section,  is there a reason for that?

Reply:
I wanted to make it easier for the "install + customize + commit" path.  Where "customize" could be *anything* rather than our limited postprocess support.  This was mentioned in the commit message.

The `test-installroot.sh` covers this - it doesn't do anything *interesting* in the "customize" phase, but it really could be anything as we don't e.g. turn off networking or run in any kind of enforced container at all